mlflow/langchain/__init__.py

```diff
@@ -127,6 +128,44 @@ def get_default_conda_env():
     return _mlflow_conda_env(additional_pip_deps=get_default_pip_requirements())
 
 
+def _try_set_databricks_auth_env_vars():
+    try:
+        from databricks.sdk import WorkspaceClient
+
+        w = WorkspaceClient()
+        if "DATABRICKS_HOST" not in os.environ:
+            os.environ["DATABRICKS_HOST"] = w.config.as_dict()["host"]
+        if "DATABRICKS_TOKEN" not in os.environ:
+            os.environ["DATABRICKS_TOKEN"] = w.config.authenticate()["Authorization"][7:]
+    except Exception as e:
+        logger.debug(f"Failed to set Databricks environment variables: {e}")
```

Comment:
Instead of mutating `os.environ`, can we pass `DATABRICKS_HOST` and `DATABRICKS_TOKEN` as parameters to the subprocess?

Reply:
If this fails and users need to manually set them, do we need to add another parameter `extra_env_vars` for users to pass them when logging the model?

Reply:
can you elaborate?